### PR TITLE
different ways to print hello world

### PR DESCRIPTION
**Pull request by Madhur-Jatiya**

### CODE REVIEW

File: java/WaysToPrintHello.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+//different ways to print hello world
+
+
+public class WaysToPrintHello {
+    WaysToPrintHello()
+    {
+        System.out.println("This is my Second way of Program");
```
Comment (Dimas-Saputra-Me) [inline java/WaysToPrintHello.java:7]: change this to ```System.out.println("Hello World!");```

File: java/WaysToPrintHello.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+//different ways to print hello world
```
Comment (Dimas-Saputra-Me) [inline java/WaysToPrintHello.java:1]: Please add information file as stated in the template file, on this line
```sh
// Title  : [Title] 
// Author : [Your-Profile-Username](Your-Profile-Link)
```

File: java/WaysToPrintHello.java (hunk 1 above)
Comment (Dimas-Saputra-Me) [inline java/WaysToPrintHello.java:7]: change this to ```System.out.println("Hello World!");```

**Review by Dimas-Saputra-Me — CHANGES_REQUESTED** [review]
It looks great and awesome, but I think you need to add and change something as I stated in the review page, It would be great if you change it, thank you.

File: java/WaysToPrintHello.java (hunk 2 above)
Comment (Madhur-Jatiya) [reply java/WaysToPrintHello.java:1]: ok i do

**Pull request closed without merging**